base_movement:
  ros__parameters:
    real_start_point: False
    current_position_info_period: 5
